edb/repl/__init__.py

```diff
@@ -704,6 +709,7 @@ def _command_list_object_types(
                           f'{eql_quote.quote_literal(pattern)}.')
                 elif 'system' not in flags:
                     print(R'No user-defined object types found. Try \ltS.')
+                self._command_semicolon_hint(arg)
```

Comment:
This should be in the `if pattern` block

Reply:
OK